src/class-parsely.php

```diff
@@ -492,6 +487,37 @@ public function construct_parsely_metadata( array $parsely_options, WP_Post $pos
 		return apply_filters( 'wp_parsely_metadata', $parsely_page, $post, $parsely_options );
 	}
 
+	/**
+	 * Sets all metadata values related to post time.
+	 *
+	 * @param array   $metadata Array containing all metadata. It will be potentially mutated to add keys: dateCreated, dateModified, & datePublished.
+	 * @param WP_Post $post     Post object from which to extract time data.
+	 * @return void
+	 */
+	private function set_metadata_post_times( array &$metadata, WP_Post $post ): void {
+		$date_format = 'Y-m-d\TH:i:s\Z';
```

Comment:
Could we extract this to a constant within the class?

Reply:
What benefit does that serve?

Reply:
I see this is only used inside this function. Dismiss my comment.